### DISCUSSION TITLE
About charge before two months free

### DISCUSSION THREAD

**Discussion by edgarexpert** [community/community#36909] [Copilot Conversations]
I want to understand why I was automatically charged without the two months free. How it works? If I cancel before two months will I be reimbursed?

**Comment by iamcjwilliams** [comment]
@edgarexpert Sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted

**Reply by edgarexpert** [reply]
Okay, thanks for the feedback, I created a ticket 3 days ago.

**Reply by iamcjwilliams** [reply]
Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. We will respond as soon as possible. Thank you for your patience
